app/models/collection.rb

```diff
@@ -156,7 +156,7 @@ def title
     too_long: ts("must be less than %{max} characters long.", max: ArchiveConfig.SUMMARY_MAX)
 
   validates_format_of :header_image_url, allow_blank: true, with: URI::regexp(%w(http https)), message: ts("is not a valid URL.")
-  validates_format_of :header_image_url, allow_blank: true, with: /\.(png|gif|jpg)$/, message: ts("can only point to a gif, jpg, or png file."), multiline: true
+  validates_format_of :header_image_url, allow_blank: true, with: /\.(png|gif|jpg|jpeg)$/, message: ts("can only point to a gif, jpg, jpeg or png file."), multiline: true
```

Comment:
I'm wondering if it makes sense to have both jpg and jpeg in the message, since they're the same file format. Maybe `jpg (jpeg)` or `jpg/jpeg`? The regex can also be simplified a bit:
```suggestion
  validates_format_of :header_image_url, allow_blank: true, with: /\.(png|gif|jpe?g)$/, message: ts("can only point to a gif, jpg, jpeg or png file."), multiline: true
```

Reply:
Let me try that out. regex is still a beast that I'm attempting to master.